x-pack/package.json

```diff
@@ -202,6 +203,8 @@
     "inline-style": "^2.0.0",
     "intl": "^1.2.5",
     "io-ts": "^1.4.2",
+    "jest": "^24.1.0",
```

Comment:
why packages are duplicated in "devDependencies" and "dependencies"?

Reply:
Probably a mistake. I would be surprised if we wanted to ship `jest` in the production build to users.

Reply:
Sometimes it happens. Contentful did it to run their functional tests 😅